plugins/audio_file_processor/audio_file_processor.cpp

```diff
@@ -343,7 +343,12 @@ void audioFileProcessor::setAudioFile( const QString & _audio_file,
 
 void audioFileProcessor::reverseModelChanged( void )
 {
-	m_sampleBuffer.setReversed( m_reverseModel.value() );
+	if (m_reverseModel.value () != m_isCurrentlyReversed) {
```

Comment:
it `loadSettings` calls this functions, `m_isCurrentlyReversed` won't be initialized (at least, not for the loaded buffer; it'll be initialized to `m_reverseModel`'s current value) on line 269.

Reply:
I still don't know how to solve that. Maybe just a `isLoading` attribute. :\